generator/osm2meta.cpp

```diff
@@ -326,6 +326,11 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_brand(std::string const
   return v;
 }
 
+std::string MetadataTagProcessorImpl::ValidateAndFormat_capacity(std::string const & v)
+{
+  return v;
```

Comment:
return strings::IsASCIINumeric(v) ? v : std::string();

Reply:
@biodranik wanted to keep the string as is to support extra details:

> 1. Is it possible to store a detailed capacity string instead of a number, to support all possible details?

https://github.com/organicmaps/organicmaps/pull/7026#issuecomment-1872612498

Reply:
ok